Makefile

```diff
@@ -37,11 +37,13 @@ test: node_modules ## Runs tests
 i18n-extract: ## Extract strings for translation from the source code
 	npm run mmjstool -- i18n extract-webapp
 
-node_modules: package.json package-lock.json
+node_modules: package.json package-lock.json packages/mattermost-components/package.json
 	@echo Getting dependencies using npm
 
 	node skip_integrity_check.js
 
+	npm install --workspaces
+	npm run build --workspace=packages/mattermost-components
```

Comment:
Having to build the subpackage before running `npm install` seems a bit odd to me. I would've expected it to be fine if that's build after install. That'd also remove the need for having to run `npm install --workspaces` since that's done by a regular `npm install`

Reply:
Will fix this up.